docs/reference/ilm/actions/ilm-readonly.asciidoc

```diff
@@ -0,0 +1,25 @@
+[[ilm-readonly]]
+=== Read-Only
+
+Phases allowed: warm.
+
+This action will set the index to be read-only
+(see: <<dynamic-index-settings,index.blocks.write>>)
+
+This action does not have any options associated with it.
```

Comment:
Not sure if this is needed. Either way, it's not included in the freeze action. I'd be consistent either way.